Sources/Widgets/Core/WidgetManager/index.js

```diff
@@ -194,18 +194,20 @@ function vtkWidgetManager(publicAPI, model) {
 
   publicAPI.enablePicking = () => {
     model.pickingEnabled = true;
+    model.pickingAvailable = true;
+    publicAPI.renderWidgets();
+  };
 
-    model.renderingType = RenderingTypes.PICKING_BUFFER;
-    model.widgets.forEach(updateWidgetForRender);
+  publicAPI.renderWidgets = () => {
+    if (model.pickingEnabled && model.captureOn === CaptureOn.MOUSE_RELEASE) {
```

Comment:
Maybe `updateSelectionFromXY()` should also check `model.pickingEnabled`

Reply:
`updateSelectionFromXY` has an implicit pre-condition that picking will be enabled, but it doesn't hurt to add it to be more explicit.